code/game/machinery/buttons.dm

```diff
@@ -5,11 +5,60 @@
 	desc = "A remote control switch for a mass driver."
 	var/id = null
 	var/active = 0
+	var/closed = 0
+	var/secure = 0
 	anchored = 1.0

```

Comment:
That would reduce the amount of initialized "objects" on startup.
Generally try to extend/override/implement current existing obj/datums/whatever when adding vars for use in a new obj. Only that obj will need those vars anyway and they are not uselessly there not doing anything.